src/commands/aksAccount/aksAccount.ts

```diff
@@ -106,3 +101,59 @@ export async function selectSubscriptions(): Promise<void> {
 
     await setFilteredSubscriptions(newFilteredSubscriptions);
 }
+
+export async function selectSubscription(): Promise<string | undefined> {
+    const sessionProvider = await getReadySessionProvider();
+    if (failed(sessionProvider)) {
+        window.showErrorMessage(sessionProvider.error);
+        return;
+    }
+
+    const allSubscriptions = await getAllSubscriptions(sessionProvider.result);
+    if (!allSubscriptions) return;
+
+    if (allSubscriptions.length === 0) {
+        await handleNoSubscriptionsFound();
+        return;
+    }
+
+    const quickPickItems: SubscriptionQuickPickItem[] = allSubscriptions.map((sub) => {
```

Comment:
Also note: this quick pick code is specific to GH copilot, can we please move this to aksCreateClusterFromCopilot.ts so that it is all in one file for maintainability?